### PR TITLE
new package: libcap 2.31

### PR DESCRIPTION
**Pull request by a-schaefers**
Read: https://k1ss.org/style.txt
Read: https://k1ss.org/guidestones.txt

---

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [x] I am the maintainer of this package.

### CODE REVIEW

File: community/libcap/build [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+sed -i "/SBINDIR/s#sbin#bin#" Make.Rules
```
Comment (dylanaraps) [inline community/libcap/build]: Why not just: `make SBINDIR=/usr/bin`?

File: community/libcap/build [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+sed -i "s/CFLAGS :=/CFLAGS += \$(CPPFLAGS) /" Make.Rules
+sed -i "s/LDFLAGS :=/LDFLAGS +=/" Make.Rules
+
+make KERNEL_HEADERS=/usr/include
+make prefix=/usr SBINDIR=/usr/bin lib=/lib DESTDIR="$1" RAISE_SETFCAP=no install
```
Comment (dylanaraps) [inline community/libcap/build]: 1000 https://k1ss.org/style.txt

Comment (dylanaraps) [reply community/libcap/build]: `lib=/usr/lib`

File: community/libcap/build [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+sed -i "s/CFLAGS :=/CFLAGS += \$(CPPFLAGS) /" Make.Rules
+sed -i "s/LDFLAGS :=/LDFLAGS +=/" Make.Rules
+
+make \
+    KERNEL_HEADERS=/usr/include
+
+make \
+    prefix=/usr \
+    SBINDIR=/usr/bin \
+    lib=/usr/lib \
```
Comment (dylanaraps) [inline community/libcap/build]: Not `LIBDIR`?

File: community/libcap/build [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+sed -i "s/CFLAGS :=/CFLAGS += \$(CPPFLAGS) /" Make.Rules
+sed -i "s/LDFLAGS :=/LDFLAGS +=/" Make.Rules
+
+make \
+    KERNEL_HEADERS=/usr/include
+
+make \
+    prefix=/usr \
```
Comment (dylanaraps) [inline community/libcap/build]: Not `PREFIX`?

File: community/libcap/build [hunk 5]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+sed -i "s/CFLAGS :=/CFLAGS += \$(CPPFLAGS) /" Make.Rules
+sed -i "s/LDFLAGS :=/LDFLAGS +=/" Make.Rules
+
+make \
+    KERNEL_HEADERS=/usr/include
```
Comment (dylanaraps) [inline community/libcap/build]: Why is this needed?

Comment (a-schaefers) [reply community/libcap/build]: i dont think it is
